specs/market-api.yaml

```
@@ -641,39 +590,86 @@ paths:
           to return to the Negotiation phase by sending a new Proposal.
 
         * `Cancelled` - Indicates that the Requestor himself has called
-         `cancelAgreement`, which effectively stops the Agreement handshake.
+          `cancelAgreement`, which effectively stops the Agreement handshake.
 
       responses:
-        '200':
+        200:
           description: Agreement approval result.
           content:
             application/json:
               schema:
                 type: string
                 enum: [Approved, Rejected, Cancelled]
-        '401':
-          $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '408':
-          $ref: '#/components/responses/TimedOut'
-        '409':
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        408:
+          $ref: 'common.yaml#/responses/Timeout'
+        409:
           description: Agreement not in `Pending` state.
-        '410':
-          $ref: '#/components/responses/Expired'
+        410:
+          $ref: 'common.yaml#/responses/Gone' # Expired
         default:
-          $ref: '#/components/responses/UnexpectedError'
+          $ref: 'common.yaml#/responses/UnexpectedError'
 
+  /agreements/events:
+    parameters:
+      - $ref: 'common.yaml#/parameters/pollTimeout'
+      - $ref: 'common.yaml#/parameters/maxEvents'
+      - $ref: 'common.yaml#/parameters/appSessionId'
+    get:
+      tags:
+        - requestor
+        - provider
+      operationId: collectAgreementEvents
+      summary: CollectAgreementEvents - Collects events related to an Agreement.
+      description: >
+        This is a blocking operation.
+
+        It returns events such as:
+
+        * `AgreementProposalEvent` - Noew Agreement has been proposed by a Requestor.
+
+        * `AgreementApprovedEvent` - Indicates that the Agreement has been approved by the Provider.
+          - The Provider is now ready to accept a request to start an Activity
+            as described in the negotiated agreement.
+          - The Providers’s corresponding `approveAgreement` call returns `Approved` after
+            this on the Provider side.
+
+        * `AgreementRejectedEvent` - Indicates that the Provider has called `rejectAgreement`,
+          which effectively stops the Agreement handshake. The Requestor may attempt
+          to return to the Negotiation phase by sending a new Proposal.
+
+        * `AgreementCancelledEvent` - Indicates that the Requestor himself has called
+          `cancelAgreement`, which effectively stops the Agreement handshake.
```

This is the Provider only event. 
```suggestion
        * `AgreementCancelledEvent` - Indicates that the Requestor has called
          `cancelAgreement`, which effectively stops the Agreement handshake. Only the Provider will receive this type of event
```

---

Reworded.